src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -91,12 +93,17 @@ export async function createStaticWebApp(context: IActionContext & Partial<ICrea
     } finally {
         isVerifyingWorkspace = false;
     }
-
     const client: WebSiteManagementClient = await createWebSiteClient([context, node.subscription]);
-    const wizardContext: IStaticWebAppWizardContext = { accessToken: await getGitHubAccessToken(), client, ...context, ...node.subscription };
+    const wizardContext: IStaticWebAppWizardContext = {
+        accessToken: await getGitHubAccessToken(),
+        client,
+        ...context,
+        ...node.subscription,
+        ...(await createActivityContext())
+    };
 
     const title: string = localize('createStaticApp', 'Create Static Web App');
-    const promptSteps: AzureWizardPromptStep<IStaticWebAppWizardContext>[] = [];
+    const promptSteps: AzureWizardPromptStep<IStaticWebAppWizardContext & ExecuteActivityContext>[] = [];
```

Comment:
I don't think you need this cast btw

Reply:
That's an [intersection type](https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html#intersection-types). 😄 But I think you're right, we don't need it.